include/model/NoteId.hpp

```diff
@@ -0,0 +1,9 @@
+#include <stdint.h>
```

Comment:
`<stdint.h>` - it's a **C**-std-header, please replace with `<cstdint>` **C++**-std header.

**A pro tip:** on C++ `std::` scope all headers without extensions (`.h`, `.hpp`, etc) e.g. `<vector>`, `<set>`, `<algorithm>` and so on

Reply:
done